guide/contribute/case-studies.md

```diff
@@ -0,0 +1,114 @@
+
+Most people will visit a case study to:
+* Learn how a particular problem was solved
+* Bring back or justify an idea about a particular design they would like to include in their own project
+* Understand how improvements were implemented
+* Understand the bigger insights learnt
+* Learn about the steps taken in the process
+
+Below, we will roughly outline steps you can include in your case study. This is by no means a fixed template, but merely ideas which can be put together to share your story. Feel free to change and add in additional information of steps that you feel are important to include.
+
+## How it started
+Start by sharing how this particular project/effort started.
+
+### Content ideas
+* What problem were you trying to solve?
+* How did this problem come to the source? What was the background?
+* How did the team collaborate (or perhaps this was a solo project)?
+* Share some background of the product itself
+
+### Visuals
+* Show your final design solution.
+
+## User base
+Most applications have a specific type of user or use case they are optimized for. Try to give the reader some insight as to who this is, and how the application services them. Perhaps the team did not have knowledge about their users at the beginning and did research to try to understand this.
+
+### Visuals
+* A graphic showing the user persona the application is targeting
+* A list of questions the team used to come up with the user persona
+
+Example of user persona Blixt wallet case study
+{% include picture.html
+   image = "assets/images/guide/contribute/case-studies/UserPersonasExampleBlixt.png"
+   retina = "assets/images/guide/contribute/case-studies/UserPersonasExampleBlixtX2.png"
+   mobile = "assets/images/guide/contribute/case-studies/UserPersonasExampleBlixtMobile.png"
+   mobileRetina = "assets/images/guide/contribute/case-studies/UserPersonasExampleBlixtMobileX2.png"
+   alt-text = "User personas example"
+   width = 1600
+   height = 800
+   modalWidthMobile = 400
+   modalHeightMobile = 400
+%}
+
+Example of user persona Payjoin case study
+
+
+## Methodology
+What steps led to the final solution? While the specific steps don't need to be detailed extensively, it's important to highlight the ones that provided the most valuable insights. Feel free to mention any challenges faced, key decision points regarding the chosen steps or tools.
+
+### Visuals
+* Images of userflows or snapshots of research done. Anything visual can go here that helps to tell a bit of a story. These can be messy.
+* An image showing the main insights, if you have one.
+
+## Insights
+After completing all the steps to solve the particular design challenge, you likely discovered some key insights. Share them here.
+
+### Content ideas
+* A blurb summarising what was learnt.
+* Bullet points highlighting the main insights.
```

Comment:
Both paragraphs above already talk about main insights. Feels a bit repetitive. Can we mix this up a little? I think it's fine to leave the points about insights, but each section can take a different angle. Visuals can be about annotated screenshots or other material. The Insights paragraph itself can describe those insights in detail. And content ideas can build on those insights to generate improvements.